src/cmd/build.rs

```diff
@@ -940,8 +940,13 @@ mod tests_ci_only {
 
             // then
             assert!(res.is_err());
-            assert!(format!("{:?}", res)
-                .starts_with("Err(Your wasm-opt version is 98, but we require a version >= 99."));
+            assert!(
+                format!("{:?}", res).starts_with(
+                    "Err(Your wasm-opt version is 98, but we require a version >= 99."
+                ),
+                "Expected a different output, found {:?}",
+                res
+            );
```

Comment:
This test is flaky sometimes, if the CI is re-run it succeeds. I added output of the `res` to have some more info when if fails spuriously again.